README.md

```diff
@@ -1586,7 +1586,7 @@ runtime {
 }
 ```
 
-Defaults to "us-central1-b"
+Defaults to the configuration setting `genomics.default-zones` in the JES configuration block which in turn defaults to using `us-central1-b`
```

Comment:
That's not listing the runtime attrs, that's listing workflow options. Which now that I look at it again I think it'd be weird to make a special case for this one setting (although I understand why he made that PR) considering that there are many things settable via RA and/or configuration that aren't tweakable in the workflow options